backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -315,10 +314,14 @@ public void grantCreateSchema(String user) {
   @Override
   public void setActiveUser(String username) {
     if (inTx) {
-      try {
-        jooq.execute("SET SESSION AUTHORIZATION {0}", name(MG_USER_PREFIX + username));
-      } catch (DataAccessException dae) {
-        throw new SqlMolgenisException("Set active user failed", dae);
+      if (username == null) {
+        this.clearActiveUser();
```

Comment:
why would we allow to run clearUser outside a transaction ?

Reply:
> why would we allow to run clearUser outside a transaction ( see clearActiveUser impl)?

because the if(inTx) check already ensures we are in an transaction